R/create_event.R

```diff
@@ -0,0 +1,30 @@
+#' Create a Google calendar event
+#'
+#' This function allows you to create a calendar event.
+#'
+#'
+#' @param id calendar id, obtained from [calendar_list()]
+#' @param start_time the event start time formatted "yyyy-mm-dd hh:mm:ss". Hours, minutes, seconds, optional.
```

Comment:
Is this really optional?  What happens if this is empty?

Reply:
It creates an all day event